app/pages/lab/export-workflows-dialog.jsx

```diff
@@ -36,17 +42,25 @@ class ExportWorkflowsDialog extends React.Component {
     getWorkflowsInOrder(project, { fields: 'display_name' })
       .then((workflows) => {
         this.setState({ workflows, loading: false });
-      });
-  }
 
-  toggleExport() {
-    if (this.workflowList.selectedIndex >= 0) {
-      this.setState({ workflowSelected: true });
-    }
+        // TODO: load the exports for each workflow
+        workflows.forEach((wf) => {
+          wf.get('classifications_export')
+            .then((media) => {
+              const mediaState = this.state.media;
+              mediaState[wf.id.toString()] = media[0];
+              this.setState({ media: mediaState });
+            })
+            .catch(() => {
+              // this is gonna happen, oh well
```

Comment:
This will fail with a 404 for any workflow that hasn't been exported yet. I'm not crazy about that design decision because it makes us swallow this error. It doesn't affect the user experience at all, which is why we don't do any messaging.